README.md

```diff
@@ -19,21 +19,89 @@ Automatically merging Dependabot PRs _only after some number of days_ (i.e.
 after a "quarantine" period) is one mitigation for this, which this Action
 implements.
 
-## Usage
+## Events
+
+Mergeabot's primary feature is to find Dependabot PRs that have been open for
+your configured `quarantine-days` and merge them (technically, approve and
+enable auto-merge so that status and review requirements are met.):
```

Comment:
I don't think a period is needed in this parenthetical since it is a clause and not a sentence